### ISSUE TITLE
Error when starting the game for the second time

### ISSUE THREAD

**Issue by scaronni** [RobertBeckebans/RBDOOM-3-BFG#795]
Hello,

I can run the game just fine, but after the first run I need to delete the file `~/.local/share/rbdoom3bfg/base/D3BFGConfig.cfg` or at the second run of the game I get this error:
```
WARNING: Unable to open resource file maps/_startup.resources
----- Initializing Decls -----
------------------------------
Couldn't open journal files
/proc/cpuinfo CPU processors: 4
/proc/cpuinfo CPU logical cores: 8
execing default.cfg
execing D3BFGConfig.cfg
couldn't exec autoexec.cfg
----- R_InitOpenGL -----
Initializing Vulkan subsystem
Enabled Vulkan instance extensions:
    VK_EXT_debug_utils
    VK_KHR_xlib_surface
    VK_KHR_surface
    VK_KHR_get_physical_device_properties2
Enabled Vulkan layers:
Sys_Error: Cannot find a Vulkan device that supports all the required extensions and properties.
NVIDIA GeForce GTX 1060 6GB:
  - cannot support the requested swap chain size: requested 2560x1440,  available 1920x1019 - 1920x1019
llvmpipe (LLVM 16.0.5, 256 bits):
  - cannot support the requested swap chain size: requested 2560x1440,  available 1920x1019 - 1920x1019
```
and the game refuses to start.

Deleting the configuration file and retrying, the game runs just fine.

**Comment by scaronni** [comment]
Side note, I have a GTX 1060 6GB with a G-Sync 2560x1440@144hz monitor. Regardless of the resolution (the game defaults windowed when no config file is yet present), I get the same error on the second run.

**Comment by scaronni** [comment]
Forgot:
```
# cat /proc/driver/nvidia/version 
NVRM version: NVIDIA UNIX x86_64 Kernel Module  535.54.03  Tue Jun  6 22:20:39 UTC 2023
GCC version:  gcc version 13.1.1 20230614 (Red Hat 13.1.1-4) (GCC) 
```

**Comment by scaronni** [comment]
After deleting the configuration file, the message goes on to the Vulkan layers:
```
WARNING: Unable to open resource file maps/_startup.resources
----- Initializing Decls -----
------------------------------
Couldn't open journal files
/proc/cpuinfo CPU processors: 4
/proc/cpuinfo CPU logical cores: 8
execing default.cfg
couldn't exec D3BFGConfig.cfg
couldn't exec autoexec.cfg
----- R_InitOpenGL -----
Initializing Vulkan subsystem
Enabled Vulkan instance extensions:
    VK_EXT_debug_utils
    VK_KHR_xlib_surface
    VK_KHR_surface
    VK_KHR_get_physical_device_properties2
Enabled Vulkan layers:
Enabled Vulkan device extensions:
    VK_KHR_buffer_device_address
    VK_EXT_descriptor_indexing
    VK_KHR_maintenance1
    VK_KHR_synchronization2
    VK_KHR_swapchain
Created Vulkan device: NVIDIA GeForce GTX 1060 6GB
```

**Comment by SRSaunders** [comment]
I think this is a duplicate of #783 which should be fixed by PR #787 

Could you please retest after applying this PR and report back?  Thanks.

**Comment by scaronni** [comment]
@SRSaunders it works, thank you! Game starts perfectly every time. I'll close it as dupe of #783.
